webapp/src/plugin_constants/index.ts

```diff
@@ -25,6 +25,10 @@ const ApiErrorIdSubscriptionsUnauthorized = 'subscriptions_not_authorized';
 const GeneralErrorMessage = 'Something went wrong';
 const GeneralErrorSubtitleForUser = 'Please contact your system administrator.';
 const GeneralErrorSubtitleForAdmin = 'Please check the server logs.';
+const SubscriptionAddedMsg = 'Subscription added successfully!';
+const SubscriptionUpdatedMsg = 'Subscription updated successfully!';
+const DeleteSubscriptionHeading = 'Confirm Delete Subscription';
```

Comment:
```suggestion
const DeleteSubscriptionHeading = 'Delete Subscription';
```
This is according to the Figma. Where did you get the heading that you've written here?

Reply:
![Uploading ConfirmationDialog.png…]()
To make it consistent with the mattermost's delete confirmation dialog header.

Reply:
Then, shouldn't it be "Confirm Subscription Delete"?